### PR TITLE
마이페이지 내가 쓴 글 조회

### PR DESCRIPTION
**Pull request by naekang**
- [x] 내가 쓴 글 조회
- [x] 테스트 코드

### CODE REVIEW

**Comment by naekang** [comment]
|File|Coverage [82.54%]|:green_apple:|
|:-|:-:|:-:|
|[CommentController.java](https://github.com/Gaethering/Gaethering-Server/blob/a1894fc0d690ae894b325fac7cc16459a70af1d8/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fboard%2Fcontroller%2FCommentController.java)|100%|:green_apple:|
|[MyPostInfo.java](https://github.com/Gaethering/Gaethering-Server/blob/a1894fc0d690ae894b325fac7cc16459a70af1d8/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fdto%2Fmypage%2FMyPostInfo.java)|89.04%|:green_apple:|
|[MemberServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/a1894fc0d690ae894b325fac7cc16459a70af1d8/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fservice%2Fmember%2FMemberServiceImpl.java)|89%|:green_apple:|
|[MemberController.java](https://github.com/Gaethering/Gaethering-Server/blob/a1894fc0d690ae894b325fac7cc16459a70af1d8/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fcontroller%2FMemberController.java)|59.29%|:x:|

|Total Project Coverage|64.93%|:x:|
|:-|:-:|:-:|

**Review by bokyung95 — COMMENTED** [review]
역시 빠르게 작성해주셨네요! 수고 많으셨습니다:D

File: src/main/java/com/gaethering/gaetheringserver/domain/member/dto/mypage/MyPostsResponse.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+@Getter
+@AllArgsConstructor
+@Builder
+public class MyPostsResponse {
+
+    private int postCount;
+
+    private List<MyPostInfo> posts;
+
+}
```
Comment (younghoondoodoom) [inline src/main/java/com/gaethering/gaetheringserver/domain/member/dto/mypage/MyPostsResponse.java:17]: 위 커멘트랑 같은 생각입니다!

**Review by younghoondoodoom — COMMENTED** [review]
테스트 코드까지 깔끔하게 작성 해주셨네요! 고생 하셨습니다

File: src/main/java/com/gaethering/gaetheringserver/domain/member/dto/mypage/MyPostInfo.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    private LocalDateTime createdAt;
+
+    public static MyPostInfo of(Post post) {
+        return MyPostInfo.builder()
+            .postId(post.getId())
+            .title(post.getTitle())
+            .createdAt(post.getCreatedAt())
+            .build();
+    }
+}
```
[the remark this answers is not in the document]
Comment (naekang) [reply src/main/java/com/gaethering/gaetheringserver/domain/member/dto/mypage/MyPostInfo.java:25]: 넵 알겠습니다!

**Comment by naekang** [comment]
|File|Coverage [82.54%]|:green_apple:|
|:-|:-:|:-:|
|[CommentController.java](https://github.com/Gaethering/Gaethering-Server/blob/b98246dbad0bb03cde158b24b521e89ca2b0dd18/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fboard%2Fcontroller%2FCommentController.java)|100%|:green_apple:|
|[PostInfo.java](https://github.com/Gaethering/Gaethering-Server/blob/b98246dbad0bb03cde158b24b521e89ca2b0dd18/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fdto%2Fmypage%2FPostInfo.java)|89.04%|:green_apple:|
|[MemberServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/b98246dbad0bb03cde158b24b521e89ca2b0dd18/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fservice%2Fmember%2FMemberServiceImpl.java)|89%|:green_apple:|
|[MemberController.java](https://github.com/Gaethering/Gaethering-Server/blob/b98246dbad0bb03cde158b24b521e89ca2b0dd18/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fdomain%2Fmember%2Fcontroller%2FMemberController.java)|59.29%|:x:|

|Total Project Coverage|64.93%|:x:|
|:-|:-:|:-:|

**Comment by naekang** [comment]
코멘트 내용 다 수정했고 머지하도록 하겠습니다! 😀

**Pull request merged**